src/components/widgets/Dropdown.jsx

```diff
@@ -39,9 +39,28 @@ class Dropdown extends PureComponent {
   render() {
     const { ...otherProps } = this.props;
     let dropDownValue = undefined;
+    let validations = "";
+
     if (typeof otherProps.input !== "undefined") {
       dropDownValue = otherProps.input.value;
     }
+
+    if (typeof otherProps.meta !== "undefined") {
+      const {
+        error,
+      } = otherProps.meta;
+      
+      validations =  (
+        <div className="div-error">
+          <span
+            className='field-error'
+            style={{ visibility: error ? 'visible' : 'hidden' }}
+          >
+            {error ? error : '_'}
+          </span>
+        </div>
+      );
+    }
```

Comment:
Should we extract this out to a "validations" widget that could get included in CustomDatePicker, Dropdown and FieldInput?